src/stdlib_ansi_cursor.f90

```diff
@@ -0,0 +1,107 @@
+module stdlib_ansi_cursor
```

Comment:
Thank you for reviewing!

It could be, but I thought separating it would be better since `stdlib_ansi` revolves around it's own derived type whereas this doesn't... I'm open to either way though